simulation/stress_test/prepare_stress_test.py

```diff
@@ -0,0 +1,111 @@
+import sys
+sys.path.append("../../")
+import os
+import configparser
+import jupiter_config
+import logging
+import random
+import paramiko
+import socket
+import _thread
+import multiprocessing
+from multiprocessing import Process, Manager
+from flask import Flask, request
+
+
+
+logging.basicConfig(level = logging.INFO)
+app = Flask(__name__)
+
+# This script must run on XDC to put random stress test on random nodes in order to collect stats from DCOMP testbed 
+
+def get_nodes(node_info_file):
+  """read the node info from the file input
+  
+  Args:
+      node_info_file (str): path of ``node.txt``
+  
+  Returns:
+      dict: node information 
+  """
+  nodes = []
+  homes = []
+  node_file = open(node_info_file, "r")
+  for line in node_file:
+      node_line = line.strip().split(" ")
+      if node_line[0].startswith('home'): 
+        homes.append(node_line[1])
+      else:
+        nodes.append(node_line[1])
+  return homes, nodes
+
+def build_push_stress():
+    jupiter_config.set_globals()
+    print(jupiter_config.STRESS_IMAGE)
```

Comment:
Prepend this print with something like "stress docker image:".

Reply:
fixed